docs/source/configuring_zipkin.rst

```diff
@@ -166,6 +166,21 @@ zipkin.always_emit_zipkin_headers
     about 300us on every non-traced request.
 
 
+zipkin.request_context
+~~~~~~~~~~~~~~~~~~~~~~
+    If it contains a valid request attribute, this specifies the stack
+    for storing the zipin attributes. If the name is invalid or the option
+    is missing, attributes will be stored in a thread local context.
+
+    This option enables support for an cooperative multithreading environment
+    (e.g. asyncio).
```

Comment:
It _should_ also enable support for multithreaded operation, right?

Reply:
Yes, but multithreading is also possible with threading.local, isn't it?